workers/tester.sh

```diff
@@ -4,7 +4,7 @@
 
 # This script should always run as if it were being called from
 # the directory it lives in.
-script_directory=`dirname "${BASH_SOURCE[0]}"`
+script_directory=`realpath $( dirname "${BASH_SOURCE[0]}" )`
```

Comment:
Just realized another optionthat uses a pipe:
```
script_directory=`dirname "${BASH_SOURCE[0]}"  | xargs realpath`
```
Please pick up whatever way you feel makes more sense.